### PR TITLE
Suryasaha manual loading

### PR DESCRIPTION
**Pull request by suryasaha**
Added manual loading and addjbrowse instructions

### CODE REVIEW

File: sphinx_doc/user_guide.rst [hunk 1]
```diff
@@ -24,26 +24,28 @@ Getting started
   * Use ``python manage.py createsuperuser``.
   * Follow the instructions shown on your terminal, then browse and login to the admin page of genomics-workspace. Usually, the admin page should be at ``http://127.0.0.1:8000/admin/``.
   * If you already have an admin account, use ``python manage.py runserver`` and then browse and login to genomics-workspace.
+  
+* All opeations on the command line will be done as the i5k user which is a admin user. You can switch to the i5k user with ``sudo su -l i5k``. You need to be in the apollo group to be able to run this command.
```
Comment (mpoelchau) [inline sphinx_doc/user_guide.rst:28]: Change to 'You need to be in the i5k group'

File: sphinx_doc/user_guide.rst [hunk 2]
```diff
@@ -24,26 +24,28 @@ Getting started
...
 
   * Peptide/Protein
   * Nucleotide/Genome Assembly
   * Nucleotide/Transcript
 
-* Copy all fasta files to be formatted for blast to media/blast/db
+* Copy all fasta files to be formatted for blast to /usr/local/i5k/media/blast/db/
 
-* Copy protein fasta files to be formatted for hmmer to media/hmmer/db
+* Copy protein fasta files to be formatted for hmmer to /usr/local/i5k/media/hmmer/db
```
Comment (mpoelchau) [inline sphinx_doc/user_guide.rst:43]: We can leave this here for now, but hmmer will also read from the media/blast/db directory... perhaps we should talk to @g8tor about best practices for this

File: sphinx_doc/user_guide.rst [hunk 3]
```diff
@@ -98,9 +100,14 @@ An admin user can add or remove data from the genomics-workspace database via th
 
 * ``python manage.py blast_shown [path of fasta file] -shown ‘true’`` (e.g python manage.py blast_shown media/blast/db/GCF_003254395.2_Amel_HAv3.1_genomic.fna -shown ‘true’)
 
+If you don't see the new organism under the blast webapp, you can login into the postgres database and see if the ``blast_blastdb`` table has the rows with information about your organism.
 
+6. To show the genome in jbrowse
 
-HMMER Database Configuration
+* ``python manage.py addjbrowse [genome assembly fasta file] -shown ‘true’`` (e.g python manage.py addjbrowse GCF_003254395.2_Amel_HAv3.1_genomic.fna https://apollo.nal.usda.gov/apollo/[genus]%20[species]/jbrowse/)
```
Comment (mpoelchau) [inline sphinx_doc/user_guide.rst:107]: Change to: * ``python manage.py addjbrowse [genome assembly fasta file] [Jbrowse URL]`` (e.g python manage.py addjbrowse GCF_003254395.2_Amel_HAv3.1_genomic.fna https://apollo.nal.usda.gov/apollo/[genus]_[species]/jbrowse/)

File: sphinx_doc/user_guide.rst [hunk 4]
```diff
@@ -120,18 +127,20 @@ Manually creating a HMMER database
...
+An admin user like i5k can add or remove data from the genomics-workspace database via the command line interface. Here, we describe how to use commands to interact with the database.
 
 1.	To add organism (not necessary if the organism is already added)
 
 * ``python manage.py addorganism [genus] [species]`` (e.g python manage.py addorganism Apis mellifera)
 
-2.	To add hmmer
+2.	To add hmmer database
+
+* ``python manage.py addhmmer [genus] [species] -f [path of fasta file] -d [genus] [species] [annotation name], [sequence type]`` (e.g python manage.py addhmmer Apis mellifera -f  media/blast/db/GCF_003254395.2_Amel_HAv3.1_genomic.fna -d "Apis mellifera Apis_mellifera_Annotation_Release_103, peptides")
```
Comment (mpoelchau) [inline sphinx_doc/user_guide.rst:138]: We only support protein databases - Can you change `-f [path of fasta file]` to `-f [path of protein fasta file]`?

**Review by mpoelchau — CHANGES_REQUESTED** [review]
Thanks @suryasaha, I left a couple of inline comments.

File: sphinx_doc/user_guide.rst (hunk 2 above)
Comment (suryasaha) [reply sphinx_doc/user_guide.rst:43]: Good point

File: sphinx_doc/user_guide.rst (hunk 3 above)
Comment (suryasaha) [reply sphinx_doc/user_guide.rst:107]: Oops. Copy paste vestige. Done

File: sphinx_doc/user_guide.rst (hunk 4 above)
Comment (suryasaha) [reply sphinx_doc/user_guide.rst:138]: Done

**Review by mpoelchau — APPROVED** [review]

**Pull request merged**